comp/core/flare/flare.go

```diff
@@ -116,7 +116,22 @@ func (f *flare) onAgentTaskEvent(taskType rcclienttypes.TaskType, task rcclientt
 		return true, fmt.Errorf("User handle was not provided in the flare agent task")
 	}
 
-	filePath, err := f.Create(nil, 0, nil)
+	enableProfiling, found := task.Config.TaskArgs["enable_profiling"]
+	if !found {
+		f.log.Debug("enable_profiling arg not found, creating flare without profiling enabled")
+		enableProfiling = "false"
+	}
+
+	flareArgs := types.FlareArgs{}
+	if enableProfiling == "true" {
+		// RC expects the agent task operation to provide reasonable default flare args
+		flareArgs.ProfileDuration = time.Second * 60
```

Comment:
Could we create a configuration setting for this. It doesn't have to be documented in the `datadog.yaml.example` but having a way to configure it could be useful.

Reply:
Sold